### DISCUSSION TITLE
Editable note previews?

### DISCUSSION THREAD

**Discussion by LiaTheNun** [RafaelGB/obsidian-db-folder#1055] [Ideas]
I just discovered the note preview feature and I'm extremely impressed and delighted! It's already useful without being fully editable, but if I could edit the notes using the preview window, my working efficiency would absolutely skyrocket. 

Is it extremely hard to make the preview editable?

**Comment by RafaelGB** [comment]
Thank you for your kind words about the note preview feature! I’m delighted to hear that you find it useful. Unfortunately, due to time constraints, I’m currently unable to maintain the plugin. However, the feature you mentioned is something I have planned for future versions.

Once Obsidian’s Dynamic Views functionality (which is similar to the Dataview search engine) is implemented natively, I intend to revisit the project and start from scratch with that idea in mind.
